util/sym-mode.go

```diff
@@ -0,0 +1,262 @@
+// Mgmt
```

Comment:
Please move this file to engine/util/ and name it mode.go or something without a dash in it please. (Same with test.)